test_unstructured_ingest/expected-structured-output/gcs/nested-2/stanley-cups.xlsx.json

```diff
@@ -1,96 +1,96 @@
 [
```

Comment:
why are all of the expected results getting bumped? it looks like different order of elements and datetime format? is there anything else going on? and reason for those changes?

Reply:
datetime is not being converted to an int for all fsspec connectors which lets us check if modification date from the source is newer than files that were potentially downloaded already. Also the `url` is the full remote path where the `remote_file_path` is the one passed in, so if recursive, this is the root of where the file was grabbed from. Open to either approach. Not sure why the order of elements would change, I haven't noticed that.